doc/advanced.rst

```diff
@@ -183,6 +183,23 @@ Handlers can be registered with *tokens*. A token is anything that supports equa
 
 
 
+Registration Guard
+------------------
+Sometimes registration should have a way to enable/disable it dynamically. Handlers can be registered with *guard*. A guard is callable which return if the handler should be called.
```

Comment:
Sometimes it is desirable to have a way to enable/disable registrations dynamically. For this purpose, gossip supports *guards* on handlers. A guard is a callable object, returning whether or not the handler should be triggered